x/datadeal/types/keys.go

```diff
@@ -43,6 +45,10 @@ func GetDataSaleKey(verifiableCID string, dealID uint64) []byte {
 	return append(DataSaleKey, CombineKeys(sdk.Uint64ToBigEndian(dealID), []byte(verifiableCID))...)
 }
 
+func GetDataDeliveryVoteKey(verifiableCID string, voterAddress sdk.AccAddress, dealID uint64) []byte {
+	return append(DataDeliveryVoteKey, CombineKeys([]byte(verifiableCID), voterAddress, sdk.Uint64ToBigEndian(dealID))...)
```

Comment:
As mentioned in #438 [comments](https://github.com/medibloc/panacea-core/pull/438#discussion_r987448680), I suggest to change the order.

 ```suggestion
	return append(DataDeliveryVoteKey, CombineKeys(sdk.Uint64ToBigEndian(dealID), []byte(verifiableCID), voterAddress)...)
```

Reply:
Thanks, I fix it. But I wonder what importance the order in the key has. Can you explain?

Reply:
When querying a list of items (e.g. oracles, deals, etc.), we usually use get the data based on prefix of key.

In this case, when the `dealID` comes first, we can query all the votes of a specific deal using a combination of prefix of `DataDeliveryVoteKey` and `sdk.Uint64ToBigEndian(dealID)`.

Likewise, we can also query all the votes for a specific `verifiableCID` using a combination of prefix of `DataDeliveryVoteKey`, `sdk.Uint64ToBigEndian(dealID)` and `[]byte(verifiableCID)`

Reply:
Thanks, I understand now 👍 !

Reply:
Thanks, I fix it!